init.lua

```diff
@@ -74,6 +80,18 @@ function SkyRocket:new(options)
     windowCanvas = createResizeCanvas(),
     resizeModifiers = options.resizeModifiers or {'ctrl', 'shift'},
     targetWindow = nil,
+
+    fancyZoneModifier = options.fancyZoneModifier or {'shift'},
```

Comment:
I am concerned that `shift` might be too few modifiers by default. Can we maybe do `{'alt', 'shift'}` for the fancy resizer?

Reply:
(you are of course welcome to use only `shift` if that works best for you!)